assets/default_config.ini

```diff
@@ -146,26 +146,40 @@ detect_sub_challenge_spam = False
 
 [advanced-filtering]
 #--------------------------------------------------------------------------------
-	# The filtering modes where it will also check for non-matched users that posted several duplicate comments
-	# If a commenter has already been marked as a match, their comments will not be shown as duplicates anyway
+	# This will check for multiple similar comments by the same author in the specified filtering modes.
+	# If a commenter has already been marked as a match, all their comments will be removed and won't show here
 	# For multiple, must be comma separated list. To disable duplicate scanning altogether, enter only 'None'
 	# NOTE: Currently does not work for Community Posts and Entire Channel Mode
 	# Default = AutoSmart, SensitiveSmart  --  Possible Values:  None | ID | Username | Text | NameAndText | AutoASCII | AutoSmart | SensitiveSmart
 duplicate_check_modes = AutoSmart, SensitiveSmart
 
-	# Determines at least how similar two comments must be to be considered a duplicate
-	# 1.0 is 100% the same, 0.5 is 50% similar, and so on. Read more here: https://en.wikipedia.org/wiki/Levenshtein_distance
-	# If you enter 0.0, it will simply flag all comments from users who left more comments than the minimum_duplicate threshold
-	# Default = 0.9 | Possible Values: 0.00 to 1.00
-levenshtein_distance = 0.9
-
 	# The minimum number of duplicate/similar comments a single user must leave before they are flagged / counted as a match
 	# The threshold is 'greater than or equal to' the number entered.
 	# NOTE 1: At this time, if the threshold is reached for a user, all of their comments will be flagged, even if all of them aren't necessarily similar
 	# NOTE 2: Duplicates are measured per-video. Comments on different videos are not compared
 	# Default = 4  --  Possible Values: [Any integer above 2]
 minimum_duplicates = 4
 
+	# This will check if a comment was posted previously by someone else in the specified filtering mode.
+	# If a commenter has already been marked as a match, all their comments will be removed and won't show here
+	# For multiple, must be comma separated list. To disable duplicate scanning altogether, enter only 'None'
+	# NOTE: Currently does not work for Community Posts and Entire Channel Mode
+	# Default = AutoSmart, SensitiveSmart  --  Possible Values:  None | ID | Username | Text | NameAndText | AutoASCII | AutoSmart | SensitiveSmart
+repost_comment_check_modes = AutoSmart, SensitiveSmart
+
+	# The minimum length a comment must be before it is checked against other comments for similarity.
+	# SETTING THIS VALUE TOO LOW IS NOT RECOMMENED - Common messages like "Hi", "first" will be flagged
+	# The threshold is 'greater than or equal to' the number entered.
+	# NOTE 1: At this time, an author is found to be repeating other comments, all of their comments will be flagged, even if all of them aren't copying other messages
+	# Default = 24  --  Possible Values: [Any integer above 0] 
+repost_minimum_text_length = 25
```

Comment:
up for discussion i guess. 25 has served me well - on one vid with 20k comments it had a few false positives because everyone was asking the same question - "why did xx leave xx"